service/controller/v13/encrypter/kms/error.go

```diff
@@ -0,0 +1,23 @@
+package kms
+
+import (
+	"github.com/aws/aws-sdk-go/aws/awserr"
+	"github.com/aws/aws-sdk-go/service/kms"
+	"github.com/giantswarm/microerror"
+)
+
+var invalidConfigError = microerror.New("invalid config")
+
+// IsInvalidConfig asserts invalidConfigError.
+func IsInvalidConfig(err error) bool {
+	return microerror.Cause(err) == invalidConfigError
+}
+
+func IsKeyNotFound(err error) bool {
```

Comment:
Here is our error missing.

Reply:
This error comes from aws API and we are just matching here, `kms.ErrCodeNotFoundException` is already defined.

Reply:
Yes but we always have our error and its matcher. The matchers are extended at times. I would just like to keep the relation between errors and matchers consistent. That makes working with the code easier.